### PR TITLE
chore(Prettier): add settings for prettier plugin

### LINKED ISSUE #4753: Add .prettierrc file
This request is not for the library itself, but for the workspace environment.

**Is your feature request related to a problem? Please describe.**
When I use VS Code, because of there is not `.prettierrc` file, the prettier pluggin use my default settings.

**Describe the ideal solution**
Add a `.prettierrc` file, so the VSC plugin can use the correct settings to format files.

Based on the command: 
https://github.com/discordjs/discord.js/blob/e0e271162c7ee9913facbff99970177c107ef3f2/package.json#L25

### .prettierrc
```json
{
  "singleQuote": true,
  "printWidth": 120,
  "trailingComma": "all",
  "endOfLine": "lf",
  "arrowParens": "avoid"
}
```
### package.json
```js
"prettier": "prettier --config .prettierrc --write src/**/*.js typings/**/*.ts",
```

### PR DESCRIPTION
**Pull request by tguichaoua**
**Please describe the changes this PR makes and why it should be merged:**
resolve #4753 

Move prettier config into `.prettierrc` file. Allow VS Code prettier plugin to correctly format files. 

**Status**

- [x] Code changes have been tested against the Discord API, or there are no code changes
- [x] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**

- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [x] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by tipakA** [comment]
#3900 Probably still applies here

**Comment by tguichaoua** [comment]
> #3900 Probably still applies here

If it's a problem, it's possible to move config into the `package.json`.

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "singleQuote": true,
+  "printWidth": 120,
+  "trailingComma": "all",
+  "endOfLine": "lf",
+  "arrowParens": "avoid"
+}
```
Comment (Papaia) [inline .prettierrc:7]:
```suggestion
}

```

**Review by Papaia — COMMENTED** [review]
This line should be changed too
https://github.com/discordjs/discord.js/blob/e0e271162c7ee9913facbff99970177c107ef3f2/package.json#L125

File: package.json [hunk 2]
```diff
@@ -153,4 +153,4 @@
       ]
     }
   }
-}
+}
```
Comment (Papaia) [inline package.json:156]:
```suggestion
}

```

**Comment by Papaia** [comment]
> > #3900 Probably still applies here
> 
> If it's a problem, it's possible to move config into the `package.json`.

I believe you should move it to the `prettier` field in `package.json` for consistency.

**Review by iCrawl — APPROVED** [review]

**Pull request merged**